src/mapreduce.jl

```diff
@@ -316,8 +316,8 @@ _mean_denom(a, ::Type{Val{D}}) where {D} = size(a, D)
 @inline maximum(f::Function, a::StaticArray; dims=:) = _mapreduce(f, max, dims, _InitialValue(), Size(a), a)
 
 # Diff is slightly different
-@inline diff(a::StaticArray; dims) = _diff(Size(a), a, dims)
-@inline diff(a::StaticVector) = diff(a;dims=Val(1))
+@inline diff(a::StaticArray; dims=Val(1)) = _diff(Size(a), a, dims)
```

Comment:
Julia Base doesn't have a default value either: https://github.com/JuliaLang/julia/blob/c068048f06619c0cfa35d76739632a5cff14836d/base/multidimensional.jl#L1013 . Maybe it was intentional?

Reply:
Ah, then it is indeed probably intentional; will revert.